### PR TITLE
Set up locale env variables

### PR DESCRIPTION
**Pull request by lukaszzdanikowski**
Setting those variables fixes the issue that sonar has with non-asci chars in filenames

### CODE REVIEW

**Comment by lukaszzdanikowski** [comment]
Hi @dagwieers @kenhuang @simonbrandhof (sorry for pointing directly at you, I've just picked last 3 contributors) are there any chances in merging this PR?

File: Dockerfile [hunk 1]
```diff
@@ -14,6 +14,12 @@ ARG NODEJS_HOME=/opt/nodejs
 
 ENV PATH=${PATH}:${SONAR_SCANNER_HOME}/bin:${NODEJS_HOME}/bin
 
+# set up local envs in order to allow for special chars (non-asci) in filenames
+ENV LC_TYPE="C.UTF-8"
```
Comment (tom-vanbraband-sonarsource) [inline Dockerfile:18]: I believe the correct key is LC_CTYPE
```suggestion
ENV LC_CTYPE="C.UTF-8"
```

**Review by tom-vanbraband-sonarsource — CHANGES_REQUESTED** [review]
Apologies for the delay.
Can you explain why setting just `ENV LC_ALL="C.UTF-8"` is not enough? [According to the documentation](https://pubs.opengroup.org/onlinepubs/7908799/xbd/envvar.html) this variable takes precedence over all others.

File: Dockerfile (hunk 1 above)
Comment (lukaszzdanikowski) [reply Dockerfile]: @tom-vanbraband-sonarsource  sorry for the delay. Yep - you are right `LC_ALL` is enough. I've adjusted the PR content.

**Review by konrad-nowacki — APPROVED** [review]

**Comment by lukaszzdanikowski** [comment]
Hi @tom-vanbraband-sonarsource any updates on the PR?

**Review by jkowalleck — APPROVED** [review]

**Review by tom-vanbraband-sonarsource — APPROVED** [review]

**Comment by tom-vanbraband-sonarsource** [comment]
Thanks @lukaszzdanikowski!

**Pull request merged**